torchmdnet/models/tensornet.py

```diff
@@ -163,6 +169,17 @@ def __init__(
         self.activation = activation
         self.cutoff_lower = cutoff_lower
         self.cutoff_upper = cutoff_upper
+        self.additional_labels = additional_labels
+        # initialize additional methods as None if not provided, also used by module.py 
```

Comment:
I think this comment is a lie, I do not see additional methods in module.py (which is how it should be)

Reply:
Yes sorry it's model.py. It's used in the forward step in model.py to define if extra_args should be None. Anyway  I will remove it, since now all the extra_args will be passed in any case and then the assertion will be inside the forward of each model.